core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.ts

```diff
@@ -86,6 +87,7 @@ export class EditThumbnailModalComponent implements OnInit {
       height: Math.round(height),
       width: Math.round(width)
     };
+    this.changeDetectorRef.detectChanges();
```

Comment:
What does `changeDetectorRef` do and why do we need it here?

Reply:
It is used to upload the thumbnail image in one click, as the parent is still in ANgularJs, change detection can only be achieved by applying it manually and I have taken help with @ashutoshc8101 and @Radesh-kumar regarding this issue.